engine/src/main/java/org/terasology/logic/players/PlayerCharacterComponent.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2017 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.logic.players;
+
+import org.terasology.entitySystem.Component;
+
+/**
+ * This is only attached to the player entities. <br/>
+ * Used to differentiate between player characters and other character entities.
+ */
+public class PlayerCharacterComponent implements Component {
```

Comment:
I guess not. All current checks look for a `characterComponent.controller` to exist to determine if the character entity is a player (controlled by a client) or not.
According to @flo this should change as an AI/NPC can be connected to a client too.